### DISCUSSION TITLE
Quickbook Customer creation and updation reflects in local?

### DISCUSSION THREAD

**Discussion by sreenak** [ruckus/quickbooks-ruby#563] [Q&A]
@rukus I have a doubt regarding QuickBook. Now when I create  a customer or update, it can be seen in Quickbook customers list. Can I do the reverse criteria. I mean if I create/update a customer from Quickbook, will it reflect in my local?

**Comment by sreenak** [comment]
Got it via webhooks

**Comment by faisalsahil** [comment]
@sreenak  Hi,  

Can you please guide me how you create a customer? Actually not mentioned creation mechanism in guide. 


Thank you
